roles/openshift_health_checker/openshift_checks/fluentd_config.py

```diff
@@ -0,0 +1,173 @@
+"""
+Module for performing checks on an Fluentd logging deployment configuration
+"""
+
+from openshift_checks import OpenShiftCheck, OpenShiftCheckException, get_var
+
+import json
+import os
+
+
+class FluentdConfig(OpenShiftCheck):
+    """Module that checks logging configuration of an integrated logging Fluentd deployment"""
+    name = "fluentd_config"
+    tags = ["health"]
+
+    logging_namespace = None
+
+    @classmethod
+    def is_active(cls, task_vars):
+        return super(FluentdConfig, cls).is_active(task_vars)
+
+    def run(self, tmp, task_vars):
+        """Check that Fluentd has running pods, and that its logging config matches Docker's logging config."""
+
+        self.logging_namespace = get_var(task_vars, "openshift_logging_namespace", default="logging")
+        fluentd_pods, error = self.get_pods_for_component(
```

Comment:
This and other `oc` calls will only work on master, though. That's what makes this tricky. We could say to just look at the logging var to see what fluentd is supposed to have and not check what it actually has. Otherwise we're gonna have to do some really weird delegation to run `oc` against master while we're looking at everything else.